packages/ui-components/src/Input.tsx

```diff
@@ -0,0 +1,160 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
```

Comment:
this entire component will need to be moved to @polkadot/ui as it is shared between this and @polkadot/apps